### ISSUE TITLE
Chrome downloads ~7MB of data when started via Puppeteer? Any way to prevent this behavior?

### ISSUE THREAD

**Issue by canselcik** [puppeteer/puppeteer#5074]
Hi,

I am aware I am not following the issue template however I think this kind of issue is higher-level than that template is intended for and this topic is expressed more succinctly without that format.

So for the sake of brevity, I think what we are seeing is effectively a manifestation of the following:
> When I launched Google Chrome for the first time (and let it sit for a minute), 32 requests were made, and 7.26 MB of data downloaded.
> https://twitter.com/jonathansampson/status/1165493206441779200

Is there any way to override this? Or use the request intercept API to intercept the requests and respond with a 404 or cache them locally? Maybe there is something for this like the undocumented `Page.setDownloadBehavior`.

Let me know what you think.

Thanks.

**Comment by benjamingr** [comment]
@canselcik puppeteer downloads and uses (by default) Chromium directly (the open source variant) rather than Chrome (the google vendored one with the addons).

So you will find it phoning to base a lot less if at all.

**Comment by canselcik** [comment]
@benjamingr Thanks for the reply. That's a very good idea. I had locked myself into using Chrome due to certain DRM issues that I had had when I was using Chromium for this automation task however I have three different kinds of instances running. Only one needs the DRM features while the other two don't.

I have just switched to using Chromium on those two kinds of instances and things seem to be working. I will run them this way for a while and see if I notice a drop in that initial phone home stage. So far there is definitely less phoning home, which is very promising.

Thanks again.
